src/main/resources/application.yaml

```diff
@@ -205,7 +205,7 @@ darts:
       response-cleanup-buffer-days: 0
       event-date-adjustment-years: 100
 
-      batch-size: ${ARM_BATCH_SIZE:10}
+      batch-size: ${ARM_BATCH_SIZE:0}
```

Comment:
Intentional?

Reply:
yes thats to switch from batch mode to single mode. There is a ticket in the next sprint to remove this into the automated tasks table.